dropwizard-metrics-influxdb/src/main/java/com/izettle/metrics/dw/InfluxDbReporterFactory.java

```diff
@@ -234,6 +241,9 @@
         .add("jvm.memory.pools.PS-Survivor-Space.usage")
         .build();
 
+    @NotNull
+    private String tagsTransformer = ClassBasedTransformer.class.getCanonicalName();
```

Comment:
Might be too much here but there "are ways" to get polymorphic support for `Transformer` here using ObjectMapper magic metadata stuff, actually I think using like the `@JsonTypeName` annotation above.